content/building/build-versioning.md

```diff
@@ -46,3 +46,141 @@ set -x
 cd $FCI_BUILD_DIR/ios
 agvtool new-version -all $(($BUILD_NUMBER + 1))
 ```
+
+## Use Codemagic CLI Tools to get the latest build number from App Store or TestFlight
+
+Use [get-latest-app-store-build-number](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/app-store-connect/get-latest-app-store-build-number.md#get-latest-app-store-build-number) or [get-latest-testflight-build-number](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/app-store-connect/get-latest-testflight-build-number.md#get-latest-testflight-build-number) actions from [app-store-connect](https://github.com/codemagic-ci-cd/cli-tools/tree/master/docs/app-store-connect#app-store-connect) Codemagic CLI Tool to get the latest build numbers.
+
+In order to do that, you need to provide API access to App Store Connect API by providing `ISSUER_ID`, `KEY_IDENTIFIER` and `PRIVATE_KEY` as arguments to the action, as defined below.
+
+Additionally, you will need to provide the Application Apple ID (an automatically generated ID assigned to your app, e.g. `1234567890`).
+It can be found under **General > App Information > Apple ID** under your application in App Store Connect.
+
+### Creating the App Store Connect API key
```

Comment:
There should be a prominent note that if they have set up automatic code signing with App Store Connect API keys in the yaml files, then they don't need to configure this again

Reply:
added it [here](https://github.com/codemagic-ci-cd/codemagic-docs/pull/267/files#diff-f6e40a5795c4d7de47d17ed5b2850ac8a06b55f09f22d3bf4611e3a4bfbffe4bR59)